doc/user/content/ops/troubleshooting.md

```diff
@@ -13,27 +13,17 @@ This page describes several SQL queries you can run to diagnose performance
 issues with Materialize.
 
 {{< warning >}}
-The introspection sources and views used in the queries below are subject to
-future changes without notice.
+The queries below reference [introspection relations](/sql/system-catalog/mz_internal/#replica-introspection-relations), which are subject to future changes without notice.
 
-This is why these sources and views are all created in the `mz_internal` schema,
+This is why these relations are all created in the `mz_internal` schema,
 and why it is not possible to create higher-level views depending on them.
```

Comment:
```suggestion
All relations below are created in the `mz_internal` schema and it is not possible to create higher-level views dependent on them.
```

Reply:
The thing we'd like to convey here is "sorry, you can't create persistent views for these troubleshooting queries because the relations they read from are unstable". So it would be nice to include a reference to the previous paragraph somehow (like the "this is why...").

Reply:
I understand the context now, thanks for the explanation. Please check out the new suggestion and make sure it conveys the correct meaning.

Reply:
It does now, thanks!